include/misc/exception.hpp

```diff
@@ -39,7 +39,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #else
 #   define EXCEPTION_HPP_NORETURN_MACRO [[noreturn]]
 #endif
-
+#if 0
```

Comment:
Probably something more specific than `0`. I don't know on which platforms this code was necessary, and I didn't end up needing it, and it was interfering with compilation.